lib/ui/screens/auth/login_screen.dart

```diff
@@ -68,10 +68,10 @@ class _LoginScreenState extends State<LoginScreen> {
                       height: h*0.05,
                     ),
                     CircleAvatar(
-                      radius: w*0.16,
+                      radius: w * 0.15,
                       child: ClipRRect(
-                        child: const Text("Login Logo"),
-                        borderRadius: BorderRadius.circular(w*0.4),
+                        child: Image.asset('assets/images/person.png'),
```

Comment:
PNGs are super expensive and require all different sizes for different sceren size. Lets use svg instead
SvgPicture.asset('assets/statusIcons/${icon}.svg')

Lets use something like this everywhere
https://pub.dev/packages/flutter_svg